code/_onclick/click.dm

```diff
@@ -63,16 +63,16 @@
  * * [obj/item/proc/afterattack] (atom,user,adjacent,params) - used both ranged and adjacent
  * * [mob/proc/RangedAttack] (atom,modifiers) - used only ranged, only used for tk and laser eyes but could be changed
  */
-/mob/proc/ClickOn( atom/A, params )
+/mob/proc/ClickOn( atom/A, list/modifiers)
```

Comment:
Why did we change it to modifier instead of params? 

Was it so `ai_interact` can pass a list if modifiers? If so, why not just convert from list to params in `ai_interact`?

Reply:
Oh I see Ghommie requested you do that, I disagree with that. I think it's preferable to convert to params in ai interact than to convert everything else to modifiers